mozilla_addon_signer/cli.py

```diff
@@ -189,6 +190,26 @@ def sign(src, dest, addon_type, api_key, attach, bucket_name, env, profile, verb
     else:
         output('\n{}'.format(json.dumps(data, indent=2, sort_keys=True)))
 
+    ctx.invoke(check_needinfo, bug_number=attach, **kwargs)
+
+
+@cli.command()
+@click.option('--api-key', '-k', default=None, help='The Bugzilla API key to use.')
+@click.argument('bug_number', nargs=1)
+def check_needinfo(bug_number, api_key):
```

Comment:
nit: would be good to have some help text

Reply:
Done :+1: